strapi-server.js

```diff
@@ -1,3 +1,3 @@
 'use strict';
 
-module.exports = require('./server');
+module.exports = require('./dist/server');
```

Comment:
Github Actions is going on about linting errors for this rule.
It's because the `dist` doesn't exist yet when we run linting in the pipeline.
You could resolve that by adding an extra step in the pipeline that does a build.
Then the linting step should require that to finish before it can run.